modules/templates/DIRECT/menus.py

```diff
@@ -227,13 +227,13 @@ def inv():
 
         return M()(
                     M("Warehouses", c="inv", f="warehouse")(
-                        M("Create", m="create"),
-                        M("Import", m="import", p="create"),
+                        M("Create", m="create", restrict=['ADMIN', 'ORG_ADMIN', 'SUPPLY_COORDINATOR']),
```

Comment:
We use double quotes in Python (and single quotes in Javascript), for consistency.